### DISCUSSION TITLE
How to choose iraf distribution? NOIRLab Iraf or Iraf community?

### DISCUSSION THREAD

**Discussion by Firestar-Reimu** [iraf-community/iraf#358] [General]
I found that apart from Iraf community, there is also a iraf distribution: https://iraf.noirlab.edu/

And it has updated to v2.18

I want to know what is the difference between NOIRLab Iraf and community Iraf

and how to choose between them

PS: it seems that https://iraf.net/ also maintains a iraf distribution but I didn't find it.

**Comment by olebole** [comment]
Cc @noirlab-iraf-admin @mjfitzpatrick from NOIRLAB

The version released by NOIRLAB is an independent fork which re-implements some (but far not all) changes that are in the community version 2.17.1. Its scope is limited to the legacy Gemini data reduction pipeline. Many bugs in their version remain unfixed, despite of the (politically chosen) higher version number. The main advantage of the NOIRLAB version is that it comes with an installer for macOS, which is still not yet available for the community version.

I have the hope that we can unify by merging their changes into the community version, but on the longer term this also depends on the will of NOIRLAB. While they were working on their version we contacted them and offered cooperation, but they refused at that time.
